string[] servo_names
float64[] positions
